spec/haml_lint/linter/spaces_for_indentation_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe HamlLint::Linter::SpacesForIndentation do
+  include_context 'linter'
+
+  # rubocop and overcomit HardTab checks don't like tabs either, but they don't
+  # check for hex, so use that to avoid triggering them
+  let(:tab) { "\x09" }
```

Comment:
The `\t` escape sequence allows us to write tabs in Ruby without using literal tab characters, and is a little more readable.